tasks/uk/census/wrapper.py

```diff
@@ -0,0 +1,132 @@
+from collections import OrderedDict, defaultdict
+import itertools
+
+from tasks.meta import current_session
+from tasks.util import MetaWrapper, TableTask
+from tasks.uk.cdrc import OutputAreas, OutputAreaColumns
+from tasks.uk.census.metadata import CensusColumns
+
+from .ons import ImportUK, ImportEnglandWalesLocal
+from .scotland import ImportScotland
+from .metadata import parse_table, COLUMNS_DEFINITION
+
+
+class Census(TableTask):
+    # Which task to use to import an specific table
+    REGION_MAPPING = {
+        "UK": ImportUK,
+        "EW": ImportEnglandWalesLocal,
+        "SC": ImportScotland
+    }
+
+    def requires(self):
+        deps = {
+            'geom_columns': OutputAreaColumns(),
+            'data_columns': CensusColumns(),
+        }
+        for t in self.source_tables():
+            deps[t] = self.task_for_table(t)
+        return deps
+
+    def timespan(self):
+        return 2011
+
+    def columns(self):
+        cols = OrderedDict()
+        input_ = self.input()
+        cols['GeographyCode'] = input_['geom_columns']['oa_sa']
+        cols.update(input_['data_columns'])
+        return cols
+
+    def source_tables(self):
+        return set(itertools.chain(*[[d['table'] for d in col['data']] for col in COLUMNS_DEFINITION.values()]))
```

Comment:
I'd simplify this expression doing more lines. Is very hard to read with multiple nested fors

Reply:
Nah, this is perfectly readable xDD. Yeah, two nested list comprehensions is too much.

Reply:
Au contraire https://www.python.org/dev/peps/pep-0020/ is non-pythonic. Cleverish is bad!